package.json

```diff
@@ -68,7 +68,7 @@
     "@babel/cli": "7.11.5",
     "@babel/core": "7.11.5",
     "@cozy/codemods": "^1.9.0",
-    "@material-ui/core": "4.11.4",
+    "@material-ui/core": "4.12.3",
```

Comment:
Since this is a breaking change, the commit should be tagged as being a breaking change. https://github.com/cozy/cozy-guidelines#commit-messages

Reply:
Updated though I'm not sure how to provide a migration path. The updated peerDependency should take care of it. Worst case scenario, everything still works but the thrown error still shows in the console.

Reply:
Worst scenario = crash since createTheme doesn't exist in previous version. 

This is not because you update the peerDep that the app consuming will automatically updates its dep. It'll throw a warning in the terminal saying "unmeet peerdep" that's all. And since there is a lot of message like this currently, app's dev will not see it.

Reply:
I think the simplest way to handle that would be to have material-ui as a real dependency. Otherwise I guess the best move is to upgrade cozy-ui to a new major version since the breaking change doesn't "fix itself" (the consumer will have to update his own material ui dep).

Reply:
Since you put a BREAKING CHANGE in the commit message, it'll publish a new major version ;) 

https://github.com/cozy/cozy-ui/issues/1865

Reply:
Ah that's nice 😅 I updated the commit message to reflect what action is necessary after updating

Reply:
Pas de souci avec la 4.12.3 ? Sur la doc https://material-ui.com/getting-started/installation/ on est encore en 4.12.2...

Reply:
J'avoue que sur les patchs en général je pars du principe que c'est sans risque, même sur les minor en fait, mais comme vous préférez j'ai juste get @latest

Reply:
Ok. Et pas d'autre BC entre la 4.11 et la 4.12 que ce que tu as modifié ?